protos/perfetto/config/priority_boost/priority_boost_config.proto

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto2";
+
+package perfetto.protos;
+
+// Configuration that allows to boost the priority of the 'traced' or
+// 'traced_probs' processes, by changing the scheduler configuration.
+// Only supported on Linux and Android the boosted process must
+// have 'CAP_SYS_NICE' capability (be started as root).
+message PriorityBoostConfig {
+  enum BoostPolicy {
+    POLICY_SCHED_OTHER = 0;
+    POLICY_SCHED_FIFO = 1;
+  };
+  required BoostPolicy policy = 1;
+  required uint32 priority = 2;
+
+  //  oneof boost_level {
+  //    // From [-20 to 19], larger values correspond to _lower_ scheduling
+  //    // priority. Userspace policy SCHED_OTHER will be used.
+  //    int32 nice_value = 1;
+  //    // From [1 to 99], lager value correspond to _higher_ priority.
+  //    // Realtime policy SCHED_FIFO will be used.
+  //    uint32 realtime_priority = 2;
+  //  }
```

Comment:
remove these an dupdate the comment son the above?

Reply:
Fixed.